src/doc/markdown/page92_customer_getting_started_with_simulator.md

```diff
@@ -0,0 +1,173 @@
+# Customer Getting Started With Simulator
+
+Note: The JaiaBot environment works __exclusively with Ubuntu 20.04__ 
+
+## Clone JaiaBot Repo On Local Machine
+
+This needs to be done in __your home directory__. For example if your user name is janesmith, the JaiaBot repo should be cloned in your home folder located at "/home/janesmith".
+
+```
+# cd /home/janesmith
+git clone https://github.com/jaiarobotics/jaiabot.git
+cd ~/jaiabot/scripts
+./setup-tools-build.sh
+./setup-tools-runtime.sh
+cd ..
+```
+
+## Install Dependencies
```

Comment:
This is a little confusing because running the scripts above does this for you. I would remove this section/adapt by putting the setup-tools-build.sh and setup-tools-runtime.sh instead of listing them like you did. The customer can always cat the file if they are interested in what dependencies are being installed.